response.go

```diff
@@ -12,12 +12,14 @@ type Header struct {
 
 func NewResponse(body interface{}) *Response {
 	return &Response{
-		headers: make([]Header, 0),
+		headers: make(map[string]string),
 		body:    body,
 	}
 }
 
 func (res *Response) Headers(headers ...Header) *Response {
```

Comment:
I would say have a type `Headers` at here would be better. Just use it like:
```go
rocket.Headers{
    "Content-Type": "application/json",
    "Allow-Origin": "*",
}
```

Reply:
Yes, It looks better, but I need to think the other name, `rocket.Headers` already used.

Reply:
I think use `Header` is ok, or create a sub pkg: `header`

Reply:
And I have a question, when would we need to read the header manually? Except content-type?

Reply:
like CSRF token, Authorization or any custom header.